### ISSUE TITLE
List plugins from which Gutenberg blocks are actually used

### ISSUE THREAD

**Issue by strarsis** [WordPress/gutenberg#14733]
**Is your feature request related to a problem? Please describe.**
After going through lots of plugins to get the right (non-core) Gutenberg blocks,
one wants to remove all unused plugins.
It would be great if there could be a list of all plugins whose Gutenberg plugins are 
actually used on the pages/posts/etc. so these can be left installed + activated.

**Describe the solution you'd like**
List of plugins whose Gutenberg blocks are actually used on the site.

**Describe alternatives you've considered**
Manually going through all pages/posts (which got Gutenberg editor) and 
noting the Gutenberg blocks down, 
then finding out the plugins that provide these Gutenberg blocks.

**Comment by swissspidy** [comment]
Sounds more like something you should report at https://core.trac.wordpress.org/, as this would require changes in WordPress that do not directly affect the Gutenberg editor itself.

Worth noting that this pretty much depends on #13693, and even then this would become quite tricky/inefficient because one would have to scan _all posts_ to see which blocks are being used.

**Comment by strarsis** [comment]
My current workaround is to use the post (page) search in backend and search for the internal block names, e.g. `advgb/table` for Advanced Gutenberg table plugin (that allows merging cells) fo find all posts (pages) that contain these Gutenberg blocks.

**Comment by mtias** [comment]
Closing in favor of #16521.
